### ISSUE TITLE
error TS2307: Cannot find module '@sentry/node/build/types/integrations/anr/common' or its corresponding type declarations.

### ISSUE THREAD

**Issue by andersk** [getsentry/sentry-electron#957]
### Is there an existing issue for this?

- [x] I have checked for existing issues https://github.com/getsentry/sentry-javascript/issues
- [x] I have reviewed the documentation https://docs.sentry.io/
- [x] I am using the latest SDK release https://github.com/getsentry/sentry-javascript/releases

### How do you use Sentry?

Sentry Saas (sentry.io)

### Electron SDK Version

5.3.0

### Electron Version

31.3.1

### What platform are you using?

Linux

### Steps to Reproduce

```shell
npm i @sentry/electron electron typescript
echo 'import * as Sentry from "@sentry/electron/main"' > test.ts
npx tsc --module esnext --moduleResolution bundler test.ts
```

### Expected Result

No errors.

### Actual Result

```
node_modules/@sentry/electron/esm/main/integrations/anr.d.ts:4:64 - error TS2307: Cannot find module '@sentry/node/build/types/integrations/anr/common' or its corresponding type declarations.

4 export declare const anrIntegration: (options?: Partial<import("@sentry/node/build/types/integrations/anr/common").AnrIntegrationOptions> | undefined) => import("@sentry/types").Integration;
                                                                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


Found 1 error in node_modules/@sentry/electron/esm/main/integrations/anr.d.ts:4
```

This is a regression in 5.0.0-beta.0.

**Comment by timfish** [comment]
Thanks for reporting this. I've managed to reproduce it in isolation but couldn't create a failing test in this repository so need to work out what exactly causes this!

**Comment by timfish** [comment]
When [this issue](https://github.com/getsentry/publish/issues/4251) closes, 5.4.0 will be published to npm!
